redaxo/src/core/lib/sql/table.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+/**
+ * Class to represent sql tables.
+ *
+ * @author gharlan
+ *
+ * @package redaxo\core
+ */
+class rex_sql_table
+{
+    use rex_instance_pool_trait;
```

Comment:
Ich verwende es ja in `get()`. 
Oder würdest du allgemein die Instanzen nicht cachen?